modules/whatsInStandard.js

```diff
@@ -0,0 +1,45 @@
+const rp = require("request-promise-native");
+const _ = require("lodash");
+const Discord = require("discord.js");
+
+class WhatsInStandard{
+    constructor() {
+        this.api = "http://whatsinstandard.com/api/4/sets.json";
+        this.commands = ["standard"];
+    }
+
+    getCommands(){
+        return this.commands;
+    }
+
+    generateEmbed(setList){
+        const currentDate = new Date();
+        const removedFutureSetList = setList.filter(set=>{
+            const releaseDate = new Date(set.enter_date);
+            return currentDate.getTime()>=releaseDate.getTime();
+        });
+        const groupedSetList = _.groupBy(removedFutureSetList,'rough_exit_date');
+        let description = "";
+        _.forEach(groupedSetList,(value,key)=>{
+           description = description.concat('*Rotates '+key+':*\n ```'+value.map(set=>set.name).join(' | ')+'```\n\n');
```

Comment:
`description = description.concat('...')` can be simplified to `description += '...'`, or, if you want to get really fancy, you could simply do `groupedSetList.map(value, key => '...').join('\n\n')` and skip the whole `forEach` block.

Reply:
After realising that goupedSetList is a hashmap, you'll have to use `_.map(groupedSetList, ...)` instead, but the rest should be the same. :)